src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -92,9 +92,9 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
                     bodyPopulatedFromFormattedMessage = true;
                 }
 
-                if (logRecord.StateValues != null)
+                if (logRecord.Attributes != null)
                 {
-                    foreach (var stateValue in logRecord.StateValues)
+                    foreach (var stateValue in logRecord.Attributes)
```

Comment:
`foreach (var attribute in ...`

Reply:
Updated. One note here. `LogRecord` now has `Body`. In order to keep this PR small (LOL!) I didn't update this logic to use it yet. Was going to do that in a follow-up.